README.md

```diff
@@ -16,8 +16,14 @@ With skaffold
 make test
 ```
 
-With docker-compose
+With docker-compose or locally
 
 ```
-SUBSTRAT_CONFIG_FILE=values-docker-compose.yaml make test
+SUBSTRAT_CONFIG_FILE=local-values.yaml make test
+```
+
+# Install substra cli
+
+```shell
+$> pip install -e ../substra
```

Comment:
The path to the substra dir depends on how you've checked out the repo. We could just say that the user will need to have the library installed and provide a link to the library's repo, that would always work. And once substra is available on pypi, we'll just update requirements.txt so that it includes the package..

Reply:
Yes let's do that